less/variables.less

```diff
@@ -166,6 +165,20 @@
 @headings-line-height:    1.2;              //1.1;
 @headings-color:          @brand-navy-minus-10;
 
+
+@display1-size:       6rem;
+@display2-size:       5.5rem;
+@display3-size:       4.5rem;
+@display4-size:       3.5rem;
+
+@display1-weight:     300;
+@display2-weight:     300;
+@display3-weight:     300;
+@display4-weight:     300;
```

Comment:
Talk to Julian about font-weight, I suspect we will match our existing headings.